peachjam/js/components/DocDiffs/ProvisionDiffInline.vue

```diff
@@ -88,21 +89,52 @@ export default {
     }
   },
   data: () => ({
+    originalElement: null,
+    wrapperElement: null,
     sideBySide: true,
     diffsets: [],
-    diffset: null
+    diffset: null,
+    vw: Math.max(document.documentElement.clientWidth || 0, window.innerWidth || 0)
   }),
 
+  watch: {
+    vw: {
+      immediate: true,
+      handler (newVw) {
+        // Turn off side by side in mobile view
+        if (newVw < 992) {
+          this.sideBySide = false;
+        }
+      }
+    }
+  },
+
   mounted () {
     this.loadDiffContentsets();
     this.originalElement = document.getElementById(this.provision.id);
+    this.wrapperElement = document.createElement('div');
+    this.wrapperElement.style.position = 'relative';
     if (this.originalElement) {
-      this.originalElement.style.display = 'none';
-      this.originalElement.insertAdjacentElement('beforebegin', this.$el);
+      // Hide originalElement behind ProvisionDiffInline via absolute positioning
+      this.originalElement.style.position = 'absolute';
```

Comment:
use reLayout from gutter

Reply:
@longhotsummer for this issue https://github.com/laws-africa/peachjam/issues/727
using `gutter.layoutItems()` does not work

The issue is that a gutter item cannot target its anchorElement if that anchor element has a style of `display: none` as shown in this demo
https://www.loom.com/share/09691fae383448a9a18685a3fa5fdf30

This is why I use an alternative way of hiding the originalElement.
Using `visiblity: hidden` preserves the element height while hiding the element

The reason I absolutely positioned the originalElement behind the DiffInlineProvision is so that gutter items associated with the originalElement can be on the same level as the DiffIlineProvision
![image](https://user-images.githubusercontent.com/15995210/205867550-a2c1f8f4-a587-46ad-9bb4-ba8abe9ebb81.png)

Please let me know if you would to chat through this.

Reply:
Ah, thanks for clarify, that makes a lot of sense. Let' go with this.